apps/showcase/src/components/showcase/component-replacement/component-replacement-pres.config.ts

```diff
@@ -0,0 +1,12 @@
+import type {Configuration} from '@o3r/core';
+import {computeItemIdentifier} from '@o3r/core';
+
+export interface ComponentReplacementPresConfig extends Configuration {
```

Comment:
Missing tsdoc for the exported objects and interfaces

Reply:
update done